packages/infrastructure/mongodb_atlas_identity_provider/main.tf

```diff
@@ -0,0 +1,26 @@
+terraform {
```

Comment:
This does not include the group mappings:

https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/federated_settings_org_role_mapping